src/molecules/AccountIndicator/index.js

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import Text from 'atoms/Text';
+import Line from 'atoms/Line';
+
+import styles from './account_indicator.module.scss';
+
+class AccountIndicator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { open: false };
+  }
+
+  getFirstName = (name) => {
+    if (!name) {
+      return '';
+    }
+
+    const [ first ] = name.split(' ');
+
+    return first;
+  }
+
+  getInitial = (name) => {
+    if (!name) {
+      return ''; // User silhoutte
+    }
+
+    return name[0];
+  }
+
+  handleIndicatorClick = () => {
+    const { open } = this.state;
+
+    this.setState({ open: !open });
+  }
+
+  render() {
+    const {
+      customerName,
+      customerEmail,
+      dashboardPath,
+      logoutPath
+    } = this.props;
+    const { open } = this.state;
+
+    return (
+      <div className={classnames(styles['container'])}>
+        <div onClick={this.handleIndicatorClick} className={classnames(styles['indicator'])}>
+          <div className={classnames(styles['circle-initial'])}>
+            <Text size={8} font='a' color='primary-2'>{this.getInitial(customerName)}</Text>
+          </div>
+
+          <Text size={9} font='a'>Hi, {this.getFirstName(customerName)}</Text>
```

Comment:
should this display a different message (or not display at all) incase `getFirstName` returns empty string? or else it would say "Hi,  " 🤔

Reply:
Ya definitely! Good catch. This should only display the text when they have a first name, otherwise just the icon. Changing!

Reply:
👍